src/views/authenticated/List.tsx

```diff
@@ -25,8 +27,10 @@ export function List({ data: unfilteredListItems, listPath }: Props) {
 
 	const Header = () => {
 		return (
-			<p>
-				Hello from the <code>/list</code> page!
+			<p className="Header">
+				For your ease, items are sorted by next purchase date. View and edit
```

Comment:
Nice removal of that p element letting us know what page it is! Great use of "Header" class name here, rather than get caught up in elements!